src/app/json-tree/abstract-parent-node.ts

```diff
@@ -0,0 +1,40 @@
+import { JsonPatch } from '../shared/interfaces';
+
+export abstract class AbstractParentNode {
+    diff?: JsonPatch[];
+    depth: number;
+    keys: Array<string> = [];
+    diffType: string;
+    isExpandedMap = new Map<string, boolean>();
+    diffKeys = new Map<string, boolean>();
+    patchesMap = new Map<string, JsonPatch[]>();
+    readonly pixelOffset = 10;
+    expand(key: string) {
```

Comment:
space btw properties.

Reply:
you mean return properties? In that case it's not returning something